src/components/AdminNavbar/AdminNavbar.module.css

```diff
@@ -64,13 +64,13 @@ a {
 
 .sidebar {
   z-index: 0;
-  position: relative;
+  padding-top: 50px;
+  position: fixed;
```

Comment:
These changes had resulted in the organizations not being displayed. So please revert them. 
![image](https://user-images.githubusercontent.com/55003601/125208942-3c823680-e25b-11eb-9a36-3cbe95a5a723.png)

Reply:
I guess this is Admin Navbar page so nothing is to be displayed over here.